apps/portal/src/components/pages/MagicLinkPage.js

```diff
@@ -180,20 +188,28 @@ export default class MagicLinkPage extends React.Component {
 
         return (
             <form onSubmit={e => this.handleSubmit(e)}>
-                <section className='gh-portal-section'>
+                <section className='gh-portal-section gh-portal-otp'>
                     {/* @TODO: create different input component with updated design */}
```

Comment:
Can this TODO be removed with the new styling in place or do we want to keep it as a note to switch to a more full-featured component later on?